postgres.pl

```diff
@@ -117,26 +117,15 @@
 sub majeur_mineur
 {
     my ($version) = @_;
-
     return ("HEAD", "", "") if $version eq "dev";
 
     # gestion du changement de numérotation depuis pg 10
     $version =~ /^(\d+).*/
-            or die "Version bizarre $version dans majeur_mineur\n";
+            or croak "Version bizarre $version dans majeur_mineur\n";
 
-    # c'est au moins pg10, le numéro de version ne contient que deux digits
-    if ($1 >= 10) {
-        $version =~ /^(\d+)(?:\.(.+))?$/
-            or die "Version bizarre $version dans majeur_mineur\n";
-
-        # Pour les versions 10+, on ajoute un faux deuxième digit de version à
-        # 0 pour éviter de compliquer le code partout ailleurs.
-        return ($1, 0, $2);
-    }
-
-    # Version inférieure à pg10, on calcule les 3 chiffres de version
+    # on calcule les 3 chiffres de version
```

Comment:
As said off-list, this leads to unexpected results due to how compare_versions works.  We need to keep 2nd digit to 0, otherwise ls_latest() will display 10.0. and 10.1 as different major versions.